test/service_test.cc

```diff
@@ -120,7 +121,7 @@ TEST_P(ServiceTest, Basic) {
   r->Write(request_, {});
   r->WritesDone();
   EXPECT_TRUE(r->Read(&response_));
-  ASSERT_FALSE(response_.has_error_detail());
+  ASSERT_TRUE(response_.has_error_detail());
```

Comment:
Check the error detail?

Reply:
Sure; I will address this in a follow up

Reply:
Done in https://github.com/envoyproxy/nighthawk/pull/167/commits/b7b40efce024e4f19248617e8c7dac897713b1fb, plus filed https://github.com/envoyproxy/nighthawk/issues/181 for tracking an improvement over that.